common/src/service-connectors/core/core.types.ts

```diff
@@ -17,6 +19,7 @@ export enum CoreEndpoint {
     GetGameSkillGroupProfile = "GetGameSkillGroupProfile",
     GetScrimReportCardWebhooks = "GetScrimReportCardWebhooks",
     GetUsersLatestScrim = "GetUsersLatestScrim",
+    GetUserByDiscordId = "GetUserByDiscordId",
```

Comment:
GetUserByAuthAccount does this

Reply:
[75d4d80](https://github.com/SprocketBot/sprocket/pull/155/commits/75d4d806f2c05d1361b2d1636188703a5ca83a20)